httpcache.go

```diff
@@ -28,6 +28,16 @@ const (
 	XFromCache = "X-From-Cache"
 )
 
+var cacheableResponseCodes map[int]bool = map[int]bool{
```

Comment:
`golint` would say:

```
httpcache.go:33: should omit type map[int]bool from declaration of var cacheableResponseCodes; it will be inferred from the right-hand side
```